### ISSUE TITLE
A11y_WINFORMS_Winforms-Containers_Winform-Containers_Keyboard:Keyboard focus is going in incorrect  direction 

### ISSUE THREAD

**Issue by Durgajyothi143** [dotnet/winforms#7354]
### .NET version

.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

**Issue description**

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG2.4.3;#DesktopApp;#Win11;#Benchmark;#Keyboard;

**Actual:**

keyboard focus is going in incorrect direction 

**Expected:**

Keyboard focus should go in correct direction

**User Impact:**

Keyboard users will face difficulties while navigating winform containers tab

**Steps to reproduce**

1. Open Core testing solution in VS 2019 Int Preview
2. Right click on cs.proj file and click on Build.
3.Then run the application. 
4.Main Form screen would appear.
5.Navigate to Containers_Controls:Testing the controls under the containers Tab.
6.Winforms_Containers screen would appear.

**Attachment:**
[A11y_WINFORMS_Winforms-Containers_Winform-Containers_Keyboard #7354.zip](https://github.com/dotnet/winforms/files/8989395/A11y_WINFORMS_Winforms-Containers_Winform-Containers_Keyboard.7354.zip)

**Comment by merriemcgaw** [comment]
@Olina-Zhang can you rearrange the controls so that the last groups with the buttons are in their own column and make sure that TabOrder is correct, left to right then top to bottom. 

Once you've gotten that could you check in the fix to the sample project and provide us an EXE to test with here? Thanks so much!

**Comment by Olina-Zhang** [comment]
Updated TabOrder for tested form, and please using following EXE for testing:
[AppExE.zip](https://github.com/dotnet/winforms/files/9006856/AppExE.zip)
![TabOrderUpated](https://user-images.githubusercontent.com/26474449/176341153-0b6312fd-938d-4f64-89d9-4671dcef577d.gif)

**Comment by merriemcgaw** [comment]
Assigning to @Durgajyothi143 to validate and close this bug.

**Comment by Durgajyothi143** [comment]
@merriemcgaw 
verified this issue in the latest environment. Issue is not fixed and still able to repro. Could you please provide latest with fixes available to verify the issue.

**Comment by merriemcgaw** [comment]
@Durgajyothi143 - did you use the zip file that @Olina-Zhang  provided with the latest .NET 7 preview installed? Can you record a gif and show me what the issue is, because it looks above like @Olina-Zhang fixed the issue you mentioned.

**Comment by merriemcgaw** [comment]
@Olina-Zhang can your team please take a look at the video provided and make sure that we've got the correct focus order in the sample again? It looks like we may have missed something. Thanks!

**Comment by Olina-Zhang** [comment]
@Durgajyothi143 The app you used is not the updated one, could you please use the above AppExE.zip I attached to test again?

**Comment by Olina-Zhang** [comment]
> Updated TabOrder for tested form, and please using following EXE for testing: [AppExE.zip](https://github.com/dotnet/winforms/files/9006856/AppExE.zip) ![TabOrderUpated](https://user-images.githubusercontent.com/26474449/176341153-0b6312fd-938d-4f64-89d9-4671dcef577d.gif) 

I provided AppExE.zip this time @Durgajyothi143

**Comment by Durgajyothi143** [comment]
@Olina-Zhang 
I have dwonloaded the zip file which u have provided but in this which file i have to open 
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/108055767/179722122-fde1689b-cb98-456f-a0e0-0cc7753e65d9.png)

**Comment by Olina-Zhang** [comment]
@Durgajyothi143 that ExE is an executable file, could you please double-click it to run in file explorer?

**Comment by Durgajyothi143** [comment]
@Olina-Zhang 
When we are trying to run "Accessibility_Core_App.exe" we are getting below error but when we try to install the version mentioned for .NET we are getting version 7.0.0-preview.6.22330.3, so could you please help us to resolve this issue



![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/108055767/180184473-fb6c95a2-3cef-427a-bd8a-0572baefe580.png)

**Comment by Olina-Zhang** [comment]
@Durgajyothi143 The issue you met is caused by .Net 7.0 version on your system is lower than that "Accessibility_Core_App.exe" app was built. Please install the higher .Net 7.0 version from: https://github.com/dotnet/installer to test. 
![image](https://user-images.githubusercontent.com/26474449/180345300-60dc3175-8a73-40fa-b0a6-b7dfcbb22ce3.png)

**Comment by Durgajyothi143** [comment]
@Olina-Zhang 
The mention version is not available in the link you have mentioned.
![Screenshot (50)](https://user-images.githubusercontent.com/108055767/180972900-a03194cf-683b-4130-abf5-385ee84fa7a5.png)

**Comment by merriemcgaw** [comment]
You can find it on this link @Durgajyothi143 
https://github.com/dotnet/installer#table

Preview 7 should be out soon but the latest builds are in the table I linked.

**Comment by Durgajyothi143** [comment]
Verified the bug as per above comment issue is not reproducing. Hence closing the bug.

[A11y_WINFORMS_Winforms-Containers_Winform-Containers_KeyboardKeyboard focus is going in incorrect direction #7354.webm](https://user-images.githubusercontent.com/108055767/182106901-2b194516-fa7b-47f9-9808-2cedea8baf4e.webm)

**Comment by Tanya-Solyanik** [comment]
Closing per request from Samhitha K S (TATA CONSULTANCY SERVICES LTD)

Issue is fixed. Adding #Closed; tag.
